modules/showDate.js

```diff
@@ -0,0 +1,14 @@
+import { DateTime } from './lexon.js';
+
+const showDate = () => {
+  const now = DateTime.now();
+  let content = '';
+  content += `
+<div id="current-date" class="p-2 float-end">
+<h3 class="text-dark">${now}</h3>
+</div>
+`;
+  return content;
+};
```

Comment:
- [ ] [OPTIONAL] It would be better if you format the date-time value to be more readable and user-friendly, as this will lead to a better UX overall, check the example below: 👇
```
const customDateFormat = {
      ...DateTime.DATE_FULL,
      ...DateTime.TIME_WITH_SECONDS,
    };
    content += `${now.toLocaleString(customDateFormat)}`;
  }
```
- You can replace the code from line `6` to line `10` with the above